.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501
```

Comment:
I would actually rather do these on a per-line basis

You can simply add `  # noqa: E501` to the end of the line that is too long to ignore that line